README.md

```diff
@@ -33,41 +26,41 @@ Output example:
 Let's assume that the ejabberd instance is named `ejabberd1`.
 
 Launch `configure-module`, by setting the following parameters:
-- `<MODULE_PARAM1_NAME>`: <MODULE_PARAM1_DESCRIPTION>
-- `<MODULE_PARAM2_NAME>`: <MODULE_PARAM2_DESCRIPTION>
-- ...
+- `hostname_certificate`: a fully qualified domain name for the Common Name of the TLS certificate, it will used as ejabberd virtualhost `user@domain.com`
+- `ldap_domain`: a Ldap domain where to authenticate the users, it could be different than hostname_certificate. The ldap_domain is used to find an existing LDAP through LDAP proxy,
```

Comment:
```suggestion
- `ldap_domain`: a Ldap domain where to authenticate the users, it could be different than hostname_certificate. The ldap_domain is used to find an existing LDAP through LDAP proxy
```